src/main/webapp/style.css

```diff
@@ -4,20 +4,13 @@
   width: 650px;
 }
 
-body{
-  background-image: url("images/Main_page_background.jpg");
-  background-color: black;
-  background-repeat: no-repeat;
-  background-attachment: fixed; 
-  background-size: cover;
-}
-
 /* Style input fields */
 label {
 	display: block;
 	padding-top: 15px;
 	padding-left: 15px;
-	font-size: 30px;
+    font-size: 30px;
+    
```

Comment:
this and below indentations does not seem consistent

Reply:
weird... on my cloud shell file it shows them lined up